### DISCUSSION TITLE
Any legal issues with packaging typst on chocolatey, winget and so on?

### DISCUSSION THREAD

**Discussion by ptsouchlos** [typst/typst#399] [General]
Is there anything that would prevent a member of the community from packaging `typst` for common package managers?

**Comment by laurmaedje** [comment]
Nope, it's already in nixOS and Arch Linux, as far as I know.

**Reply by ptsouchlos** [reply]
Perfect, I'll get on that then :)

**Reply by ptsouchlos** [reply]
For `winget`, it would be easier to generate and maintain a package if `typst` provided an installer for Windows. Is this something the team is interested in doing? It could be done as part of the `release` CI that is already set up.
